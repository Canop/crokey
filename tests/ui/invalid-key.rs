fn main() {
    crokey::key!(10);
    crokey::key!(ctrl-backpace);
    crokey::key!(ctrl--);
}
