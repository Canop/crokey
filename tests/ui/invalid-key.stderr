error: invalid key; must be between 0-9
 --> tests/ui/invalid-key.rs:2:18
  |
2 |     crokey::key!(10);
  |                  ^^

error: unrecognized key code "backpace"
 --> tests/ui/invalid-key.rs:3:23
  |
3 |     crokey::key!(ctrl-backpace);
  |                       ^^^^^^^^

error: expected one of: character literal, integer literal, identifier
 --> tests/ui/invalid-key.rs:4:23
  |
4 |     crokey::key!(ctrl--);
  |                       ^
